src/nxdoc/web-ui/web-ui-tutorials/web-ui-bulk-actions.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Bulk Actions in Web UI'
+review:
+    comment: ''
+    date: '2021-09-07'
+    status: ok
+toc: true
+labels:
+    - nuxeo-web-ui
+    - bulk
+tree_item_index: 490
+---
+
+## Enabling Bulk Actions
+
+{{#> callout type='info'}}
+Bulk actions leverage the [Bulk Action Framework]({{page page='bulk-action-framework'}}). Make sure your [architecture]({{page page='nuxeo-reference-architectures'}}) complies with it.
+{{/callout}}
+
+Bulk actions are available since LTS 2021. To prevent bringing any breaking change through an update, you need to enable this feature explicitly by adding the following property to your [nuxeo.conf]({{page page='configuration-parameters-index-nuxeoconf'}}) file:
+```
+nuxeo.selection.selectAllEnabled=true
+```
+
+## Using Bulk Actions
+
+Once enabled, users can select all documents in a results listing and execute an action on them. A selection can be done while browsing content and in the context of search results. It is possible to select all documents after using filters as well to fine tune the results first. Note that adding filters, changing the display or the sort order for the results causes the current selection to be lost: this results in sending a different query to the server and we cannot keep track of the difference between the two without hurting performance.
+
+//TODO image select all and default bulk actions
+
+//TODO image select all after filtering
+
+Clicking on the checkbox selects all documents at once, both the ones that are visible and the ones that are not currently displayed on screen. Clicking it again deselects all documents. Note that it is not possible to deselect documents individually at this stage.
+
+### Available Default Bulk Actions
+
+Once the selection is made, the following default actions are available as bulk actions:
+
+* Add to collection
+* Remove From Collection (when viewing a collection)
+* Publish
+* Delete (send to trash)
+* Untrash (in trash view)
+* Delete permanently (in trash view)
+
+The following default actions are NOT available as bulk actions:
+
+* Download all as zip. This action would not be scalable. To download files in large numbers, we recommend using [Nuxeo Drive]({{page page='nuxeo-drive'}}) instead.
+* Compare. Web UI and Nuxeo Server provide the capacity to [compare two documents]({{page space='userdoc' page='compare'}}) or versions at this stage.
```

Comment:
Compare does not work because it is done client side so there is nothing we could do with just a page provider as input. It won't work with the BAF, just like the clipboard.

Reply:
Added, thanks!